gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/Acl.java

```diff
@@ -0,0 +1,400 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import com.google.api.services.bigquery.model.Dataset.Access;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * Access Control for a BigQuery Dataset.
```

Comment:
A reference (and maybe some more description) from [here](https://cloud.google.com/bigquery/access-control)?